src/applications/edu-benefits/tests/feedback-tool/00.feedback-tool.cypress.spec.js

```diff
@@ -0,0 +1,155 @@
+
+    cy.url().should('not.contain', '/introduction');
+
+    // Applicant relationship
+    cy.get('input[name="root_onBehalfOf"][value="Myself"]', {
+      timeout: Timeouts.normal,
+    }).should('exist');
+    cy.injectAxeThenAxeCheck();
+    cy.selectRadio('root_onBehalfOf', testData.data.onBehalfOf);
+    cy.get('.usa-alert.usa-alert-info.background-color-only', {
+      timeout: Timeouts.slow,
+    }).should('be.visible');
+
+    cy.get('.form-progress-buttons .usa-button-primary').click();
+    cy.url().should('not.contain', '/applicant-relationship');
+
+    // Applicant information
+    cy.get('input[name="root_fullName_first', {
+      timeout: Timeouts.normal,
+    }).should('exist');
+    cy.axeCheck();
+
+    cy.fillName('root_fullName', testData.data.fullName);
+    cy.get('#root_serviceAffiliation').select(testData.data.serviceAffiliation);
+
+    cy.get('.form-progress-buttons .usa-button-primary').click();
+    cy.url().should('not.contain', '/applicant-information');
+
+    // Service Information
+    cy.get('select[name="root_serviceBranch"]', {
+      timeout: Timeouts.normal,
+    }).should('exist');
+    cy.axeCheck();
+
+    cy.fillDate(
+      'root_serviceDateRange_from',
+      testData.data.serviceDateRange.from,
+    );
+    cy.fillDate('root_serviceDateRange_to', testData.data.serviceDateRange.to);
+    cy.get('#root_serviceBranch').select(testData.data.serviceBranch);
+
+    cy.get('.form-progress-buttons .usa-button-primary').click();
+    cy.url().should('not.contain', '/service-information');
+
+    // Contact information
+    cy.get('select[name="root_address_country"]', {
+      timeout: Timeouts.normal,
+    }).should('exist');
+    cy.axeCheck();
+
+    cy.fillAddress('root_address', testData.data.address);
+    cy.fill('input[name="root_applicantEmail"]', testData.data.applicantEmail);
+    cy.fill(
+      'input[name="root_view:applicantEmailConfirmation"]',
+      testData.data['view:applicantEmailConfirmation'],
+    );
+    cy.fill('input[name="root_phone"]', testData.data.phone);
+
+    cy.get('.form-progress-buttons .usa-button-primary').click();
+    cy.url().should('not.contain', '/contact-information');
+
+    // Benefit information
+    cy.get('input[name="root_educationDetails_programs_chapter33"]', {
+      timeout: Timeouts.normal,
+    }).should('exist');
+    cy.axeCheck();
+
+    cy.get('#root_educationDetails_programs_chapter33', {
+      timeout: Timeouts.normal,
+    }).click();
+
+    cy.get('.form-progress-buttons .usa-button-primary').click();
+    cy.url().should('not.contain', '/benefits-information');
+
+    // School information
+    cy.get('input[type="checkbox"]', {
+      timeout: Timeouts.normal,
+    }).should('exist');
+    cy.axeCheck();
+
+    cy.get('input[type="checkbox"]', {
+      timeout: Timeouts.normal,
+    }).click();
+    cy.get(
+      'input[name="root_educationDetails_school_view:manualSchoolEntry_name"]',
+      { timeout: Timeouts.slow },
```

Comment:
How come we need to provide a Timeouts option for each of these `.get`? In theory I think Cypress will wait for long enough before failing the test 🙂

Reply:
Cypress is supposed to do that as you described.  Unfortunately, it seems in GHA we've run into some race conditions that prevent this from happening with the timing that it should. It's still something we're constantly tinkering with and trying to figure out exactly how much of this is needed, but erring on the side of caution to get GHA stable.

Reply:
I hope this won't be neccessary for _all_ of our Cypress tests in order to convert to GHA! 😨

Reply:
It's not 🙂 If you feel comfortable with it @CBonade you can try omitting the `, { timeout: Timeouts.slow/normal }` bit from each `cy.get` and if the tests still pass then we should be golden 🏅

Reply:
It definitely seems situational, and more sensitive immediately following a navigation and/or with inputs vs other elements